Directory.Build.targets

```diff
@@ -15,6 +15,11 @@
     <DefineConstants>$(DefineConstants);$(OS)</DefineConstants>
   </PropertyGroup>
 
+  <!-- Settings that allow testing to work by default -->
+  <PropertyGroup>
+    <RuntimeIdentifier Condition="'$(RuntimeIdentifier)' == ''">$(NETCoreSdkRuntimeIdentifier)</RuntimeIdentifier>
```

Comment:
The libclang package is setup using a `runtime.json` file, so tests can't run without a `RID` being specified (so the correct libclang` native dependency can be resolved). This sets the default to just be the same as the host SDK.